physics/continuous_trajectory.hpp

```diff
@@ -132,27 +132,46 @@ class ContinuousTrajectory : public Trajectory<Frame> {
   ContinuousTrajectory();
 
  private:
+  // Each polynomial is valid over an interval [t_min, t_max].  Polynomials are
+  // stored in this vector sorted by their |t_max|, as it turns out that we
+  // never need to extract their |t_min|.  Logically, the |t_min| for a
+  // polynomial is the |t_max| of the previous one.  The first polynomial has a
+  // |t_min| which is |*first_time_|.
+  struct InstantPolynomialPair {
```

Comment:
So we end up having that pair after all... Is it still worth having the reference instant replicated in the polynomial itself, or would it make sense to use polynomials of Time?

Reply:
Well, really we need a triple.  Previously it was `{t_min, t_max, series}` now it's effectively `{t_mid, t_max, polynomial}`.  I prefer the new code because there is less drudgery in `ContinuousTrajectory`.